ml/src/test/scala/ai/h2o/sparkling/ml/models/BinaryModelTestSuite.scala

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.ml.models
+
+import java.io.File
+
+import ai.h2o.sparkling.backend.BuildInfo
+import ai.h2o.sparkling.ml.algos.{H2OAutoML, H2OGBM, H2OGridSearch}
+import ai.h2o.sparkling.{SharedH2OTestContext, TestUtils}
+import org.apache.spark.sql.SparkSession
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{FunSuite, Matchers}
+
+@RunWith(classOf[JUnitRunner])
+class BinaryModelTestSuite extends FunSuite with Matchers with SharedH2OTestContext {
+
+  override def createSparkSession(): SparkSession = sparkSession("local[*]")
+
+  private lazy val dataset = spark.read
+    .option("header", "true")
+    .option("inferSchema", "true")
+    .csv(TestUtils.locate("smalldata/prostate/prostate.csv"))
+
+  private lazy val referenceAlgo = new H2OGBM()
+    .setSeed(1)
+    .setFeaturesCols("CAPSULE", "RACE", "DPROS", "DCAPS", "PSA", "VOL", "GLEASON")
+    .setLabelCol("AGE")
+
+  private val outputPath = "ml/build/binary.model"
+
+  test("Exported binary model exists") {
```

Comment:
Would it be possible to add also a negative test?

Reply:
Do you mean that if the user does not call write the model is not exported?

Reply:
Sorry, wrote the comment to a wrong test. This was meant for https://github.com/h2oai/sparkling-water/pull/2229/files#diff-d19d0cfca040eb85c975de7b1421cd4dR66. to check result of `binaryModelExists` if you don't fit the model

Reply:
Ah, yes, that is a good idea, will do!